test/example/array/demo_falseloc.f90

```diff
@@ -1,8 +1,8 @@
 program demo_falseloc
-use stdlib_array, only : falseloc
-implicit none
-real, allocatable :: array(:)
-allocate(array(-200:200))
-call random_number(array)
-array(falseloc(array < 0.5), lbound(array)) = 0.0
+    use stdlib_array, only: falseloc
+    implicit none
+    real, allocatable :: array(:)
+    allocate (array(-200:200))
+    call random_number(array)
+    array(falseloc(array < 0.5), lbound(array)) = 0.0
```

Comment:
```suggestion
    array(falseloc(array < 0.5, lbound(array,1))) = 0.0
```

This is failing in the CI, [here's the corresponding test](https://github.com/fortran-lang/stdlib/blob/master/src/tests/array/test_logicalloc.f90#L210)